<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BufferWidget</class>
 <widget class="QWidget" name="BufferWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="qvlay_composition" stretch="0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="ql_pH">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>pH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FloatingValueLineEdit" name="qle_pH">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ql_ionicStrength">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>IS (mM)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FloatingValueLineEdit" name="qle_ionicStrength">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ql_bufferCapacity">
         <property name="text">
          <string>BC (mM)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FloatingValueLineEdit" name="qle_bufferCapacity">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="qvlay_expValues" stretch="0,0,0">
     <item>
      <widget class="QLineEdit" name="qcap_expValues">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Experimental mobilities</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="qscr_expValues">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>114</width>
          <height>202</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="qvlay_expValuesInner">
       <item>
        <layout class="QHBoxLayout" name="qhlay_expValuesAdd">
         <item>
          <widget class="QPushButton" name="qpb_addExpValue">
           <property name="text">
            <string>Add mobility</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="qvlay_controls" stretch="0,0,0,0,0,0">
     <item>
      <widget class="QPushButton" name="qpb_remove">
       <property name="text">
        <string>&lt;DELETE&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_clone">
       <property name="text">
        <string>&lt;CLONE&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_export">
       <property name="text">
        <string>&lt;EXPORT&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_correctConcentration">
       <property name="text">
        <string>&lt;CORR_C&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_exclude">
       <property name="text">
        <string>&lt;EXCLUDE&gt;</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatingValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/ui/elementaries/floatingvaluelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
